src/commands/warning.js

```diff
@@ -0,0 +1,85 @@
+const { MessageEmbed } = require("discord.js");
+const schema = require('../../schemas/warnDB')
+const { colors } = require("../config/config.json");
+const pjson = require("../../package.json");
+const { re } = require("mathjs");
+
+exports.run = async (client, message, args, level) => {
+  //check for permission
+  if(!message.member.hasPermission("MANAGE_MESSAGES"))
+    {
+      return message.channel.reply("You don't have enough permission")
+    }
+    //get the user from the mention
+  let user = message.mentions.users.first()
+  if(!user) return message.channel.send("Please mention an user to warn")
+
+  //find or update the user on database
+  let data;
+  try{
+    data = await schema.findOne({
+      userId: user.id,
+      guildId: message.guild.id
+    })
+    if(!data){
+      data =await schema.create({
+        uuserId: user.id,
+        guildId: message.guild.id
+      })
+  }
+  }catch(error){
+    console.log(error)
+  }
+
+  //check the number of warning of the user
+  const warnembed = new MessageEmbed()
+    .setColor(client.config.colors.heptagram)
+    .setTitle(`:white_check_mark: **Success!** :white_check_mark:`)
+    .setDescription(
+      ` **${user.tag}** has a total of ${data.warns} warnings. `
+    )
+    .setTimestamp()
+    .setFooter({
+      text: `Message sent by Heptagram || ${pjson.version}`,
+      iconURL: `${client.config.cdn.sqlogo}`,
+
+      
+    });;
+message.channel.send(warnembed);
+
+  // const errorembed = new MessageEmbed()
+  //   .setColor(client.config.colors.heptagram)
+  //   .setTitle(`**Failed**`)
+  //   .setDescription(`Failed to warn **${user.tag}**.`)
+  //   .setTimestamp()
+  //   .setFooter({
+  //     text: `Message sent by Heptagram || ${pjson.version}`,
+  //     iconURL: `${client.config.cdn.sqlogo}`,
+  //   });
+
+//   try {
+//     await message.guild.members.warn(user, { reason });
+//   } catch (error) {
+//     return message.reply({ embeds: [errorembed] });
+//   }
+
+//   return message.reply({ embeds: [warnembed] });
```

Comment:
Why is this commented out?

Reply:
I was a bit confused if I should add the error message or not, so instead waited for your comment. **I'll go ahead and add it?**

Reply:
Sounds good! Yes, please add it.